cmd/performance-profile-creator/cmd/info.go

```diff
@@ -0,0 +1,146 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"sort"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const (
+	infoModeJSON = "json"
+	infoModeLog  = "log"
+)
+
+type infoOptions struct {
+	jsonOutput bool
+}
+
+func NewInfoCommand(pcArgs *ProfileCreatorArgs) *cobra.Command {
```

Comment:
maybe a short comment on the function since its public

Reply:
Done. Thanks